approval_polls/templates/approval_polls/edit.html

```diff
@@ -119,15 +119,15 @@ <h4 class='panel-title'>
               <div class='radio'>
                 <label class='full-width'><input type='radio' name='radio-poll-type' {% if poll.vtype == 3 %} checked='true' {% endif %} value='3'>Only specific Email owners (will be invited by Email).
                   <div id='email-input' hidden>
```

Comment:
I can display the previously entered email addresses, but can you confirm with Aaron as to what he prefers?